src/clocksi_static_tx_coord_sup.erl

```diff
@@ -28,7 +28,7 @@
 
 -export([init/1]).
 
--define(NUM, 100).
+-define(NUM, 1).
```

Comment:
I ran several tests with different numbers of supervisors on the evaluation branch. The workload is append only, so it is a little lower than mixing read with append.
In the following result, the first number is the number of supervisor, the second is the throughput, the third is the average latency.
1 3275,15 3068,58
5 3094,34 3258,11
10 3103,03 3289,53
20 2882,41 3620,37
50 3109,1 3223,42
100 3081,78 3247,63

This is the result for my own static_coord branch.
1 3461,96 2915,79
10 3539,63 2838,74
50 3522,25 2833,74
100 3488,29 2856,16